applications/StructuralMechanicsApplication/tests/membrane_test/Membrane_orthotropic_diagonal_test_parameters.json

```diff
@@ -0,0 +1,118 @@
+    },
+    "processes"        : {
+        "constraints_process_list" : [{
+            "python_module" : "assign_vector_variable_process",
+            "kratos_module" : "KratosMultiphysics",
+            "Parameters"    : {
+                "model_part_name" : "Structure.DISPLACEMENT_dirichZ",
+                "variable_name"   : "DISPLACEMENT",
+                "constrained"     : [false,false,true],
+                "value"           : [null,null,0.0],
+                "interval"        : [0.0,"End"]
+            }
+        },{
+            "python_module" : "assign_vector_variable_process",
+            "kratos_module" : "KratosMultiphysics",
+            "Parameters"    : {
+                "model_part_name" : "Structure.DISPLACEMENT_dirichX",
+                "variable_name"   : "DISPLACEMENT",
+                "constrained"     : [true,false,false],
+                "value"           : [0.0,null,null],
+                "interval"        : [0.0,"End"]
+            }
+        },{
+            "python_module" : "assign_vector_variable_process",
+            "kratos_module" : "KratosMultiphysics",
+            "Parameters"    : {
+                "model_part_name" : "Structure.DISPLACEMENT_dirichY",
+                "variable_name"   : "DISPLACEMENT",
+                "constrained"     : [false,true,false],
+                "value"           : [null,0.0,null],
+                "interval"        : [0.0,"End"]
+            }
+        }],
+        "loads_process_list"       : [{
+            "python_module" : "assign_vector_by_direction_to_condition_process",
+            "kratos_module" : "KratosMultiphysics",
+            "check"         : "DirectorVectorNonZero direction",
+            "Parameters"    : {
+                "model_part_name" : "Structure.PointLoad3D_forceX",
+                "variable_name"   : "POINT_LOAD",
+                "modulus"         : 4000000.0,
+                "direction"       : [1,0.0,0.0],
+                "interval"        : [0.0,"End"]
+            }
+        },{
+            "python_module" : "assign_vector_by_direction_to_condition_process",
+            "kratos_module" : "KratosMultiphysics",
+            "check"         : "DirectorVectorNonZero direction",
+            "Parameters"    : {
+                "model_part_name" : "Structure.PointLoad3D_forceY",
+                "variable_name"   : "POINT_LOAD",
+                "modulus"         : 4000000.0,
+                "direction"       : [0.0,1,0.0],
+                "interval"        : [0.0,"End"]
+            }
+        }],
+        "list_other_processes"     : [{
+            "python_module" : "project_vector_on_surface_process",
+            "kratos_module" : "StructuralMechanicsApplication",
+            "Parameters"    : {
+                "model_part_name"  : "Structure",
+                "projection_type"        : "planar",
+                "global_direction"       : [1,1,0],
+                "variable_name"    : "LOCAL_MATERIAL_AXIS_1"
```

Comment:
isn't it local_prestress_vector?

Reply:
not in this  case. This one checks the othrotropic claw with the membrane